src/Compilers/CSharp/Test/Symbol/Symbols/ModuleInitializersTests.Accessibility.cs

```diff
@@ -0,0 +1,392 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Symbols
```

Comment:
>namespace [](start = 0, length = 9)

Explosion of the test files feels absolutely unnecessary. If you want to keep all of them, please create a dedicated folder and move them there.

Reply:
However, postpone any movement until we finish the review process.

---
In reply to: [418144256](https://github.com/dotnet/roslyn/pull/43301#discussion_r418144256) [](ancestors = 418144256)

Reply:
Also, a dotted file name usually implies a declaration of a nested type. For example, an "Accessibility" type nested into "ModuleInitializersTests" type for this particular file name. 

---
In reply to: [418149282](https://github.com/dotnet/roslyn/pull/43301#discussion_r418149282) [](ancestors = 418149282,418144256)

Reply:
Yes, I'm familiar with that.